### PR TITLE
Modified /pom.xml and /terasoluna-gfw-parent/pom.xml for Maven Central.#664

### PR DESCRIPTION
**Pull request by bturamatuyu**
Please review #664 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,23 +30,6 @@ Two types of settings are possible.
 
 Irrespective of above two ways, first define the repositories in the pom file.
 
-(The below is required in the both the type of settings.)
```
Comment (kazuki43zoo) [inline README.md]: - [ ] Please revert this fix because it published on GitHub page immediately.

File: pom.xml [hunk 2]
```diff
@@ -34,32 +34,15 @@
...
-                <enabled>true</enabled>
-            </releases>
-            <snapshots>
-                <enabled>false</enabled>
-            </snapshots>
-            <id>terasoluna-gfw-releases</id>
-            <url>http://repo.terasoluna.org/nexus/content/repositories/terasoluna-gfw-releases/</url>
-        </repository>
-        <repository>
+            <id>sonatype-nexus-snapshots</id>
```
Comment (kazuki43zoo) [inline pom.xml]: - [ ] Is need the Sonatype Nexus Snapshots repository ?

File: README.md (hunk 1 above)
Comment (bturamatuyu) [inline README.md]: Revert is completed.

File: pom.xml (hunk 2 above)
Comment (bturamatuyu) [inline pom.xml]: Removed repositories tag in /pom.xml.

**Comment by bturamatuyu** [comment]
@kazuki43zoo san
Correspondence of comment is completed.

**Pull request merged**